bundles/org.openhab.binding.insteon/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -114,6 +115,11 @@
 		<label>Battery Level</label>
 	</channel-type>
 
+	<channel-type id="batteryPercent">
+		<item-type>Number</item-type>
```

Comment:
It is preferable for percent values to be represented as a QuantityType of `Number:Dimensionless` which uses Percent units.

Reply:
I originally tried that, but it was getting converted to a decimal number (XX.Y) instead of a whole number (XX), so I converted back to a Number.

Reply:
It should work if you make your pattern `"%d %unit%"`.

Reply:
the typical pattern is "%f.0 %unit%" but using "%d %unit%" avoids an decimal points from being printed

Reply:
batteryPercentage is a DecimalType, not a PercentType, this is due to how DeviceFeatureListener.publishState() handles PercentTypes. I'd rather just leave as is.

Reply:
`Number:Dimensionless` is a QuantityType, not a PercentType. There exists a Unit type `SmartHomeUnits.PERCENT` which is used to represent QuantityType values that are percentages.

Reply:
converted to `Number:Dimensionless`